sceptre/config/graph.py

```diff
@@ -90,15 +90,15 @@ def _generate_edges(self, stack, dependencies):
         :type dependencies: list
         """
         self.logger.debug(
-            "Generate edges for graph {0}".format(self.graph)
+            "Generate edges for graph {0}".format(stack)
```

Comment:
Now I am looking at this line, it might make sense to have something like:

```python 
self.logger.debug("Generate dependency graph for {}".format(self.graph))
```

Reply:
follow up in above conversation about the final output in debug log.